### PR TITLE
i18n(zh-cn): add `rewrite-with-body-used.mdx` and `expected-not-esmimage.mdx`

### PR DESCRIPTION
**Pull request by Nin3lee**
#### Description

Create `rewrite-with-body-used.mdx`
Create `expected-not-esmimage.mdx`
Update `error-reference.mdx`
#### Related issues & labels (optional)

- Suggested label: i18n

### CODE REVIEW

**Comment by liruifengv** [comment]
这种包含了同一个文件的两次更新，最好放到一个PR中提交。不然其中一个合并之后，`error-reference` 就会被标记为已完成。https://github.com/withastro/docs/pull/8608

**Comment by liruifengv** [comment]
还有一个新的更新，麻烦把两个PR合并到一起吧 https://github.com/withastro/docs/commit/f0c7857644096c7de1c522b1acb93c5f6a8977f9

**Comment by Nin3lee** [comment]
> 还有一个新的更新，麻烦把两个PR合并到一起吧 [f0c7857](https://github.com/withastro/docs/commit/f0c7857644096c7de1c522b1acb93c5f6a8977f9)

好的，我试试

File: src/content/docs/zh-cn/reference/errors/expected-not-esmimage.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: Expected image options, not an ESM-imported image.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **ExpectedNotESMImage**: ESM 导入的图像无法直接传递给 `getImage()`。相反，请在 `src` 属性中传递带有图像的对象。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/expected-not-esmimage.mdx:7]:
```suggestion
> **ExpectedNotESMImage**: ESM 导入的图像无法直接传递给 `getImage()`。相反，应传递一个带有图像的对象，并将图像放在 `src` 属性中。
```

File: src/content/docs/zh-cn/reference/errors/expected-not-esmimage.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+## 哪里出了问题？
+
+ESM 导入的图像无法直接传递给 `getImage()`。相反，请在 `src` 属性中传递带有图像的对象。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/expected-not-esmimage.mdx:10]:
```suggestion
ESM 导入的图像无法直接传递给 `getImage()`。相反，应传递一个带有图像的对象，并将图像放在 `src` 属性中。
```

File: src/content/docs/zh-cn/reference/errors/rewrite-with-body-used.mdx [hunk 3]
```diff
@@ -0,0 +1,21 @@
+---
+title: Cannot use Astro.rewrite after the request body has been read.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+> **RewriteWithBodyUsed**: 如果请求体已被读取，则无法使用Astro.rewrite()。如果你需要读取请求体，请先克隆请求。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/rewrite-with-body-used.mdx:7]:
```suggestion
> **RewriteWithBodyUsed**: 如果请求体已被读取，则无法使用 Astro.rewrite()。如果你需要读取请求体，请先克隆请求。
```

**Comment by liruifengv** [comment]
Thanks peeps. I have leaved some suggestions.

**Pull request merged**